notebooks/official/stable_diffusion/dreambooth_stablediffusion.ipynb

```diff
@@ -0,0 +1,1326 @@
+{
```

Comment:
<p>You might want to highlight you are doing mixed precision training,</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Done.</p>